share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -124,3 +124,33 @@ e4s-develop-build:
       - artifact: jobs_scratch_dir/cloud-ci-pipeline.yml
         job: e4s-develop-generate
     strategy: depend
+
+
+########################################
+# E4S on Power
+########################################
+.e4s-on-power:
+  variables:
+    SPACK_CI_STACK_NAME: e4s-on-power
+
```

Comment:
If generating the power pipeline on the `x86_64` image is causing problems, then right here you could add:

```
.power-e4s-generate-image:
  image: { "name": "ghcr.io/scottwittenburg/ecpe4s-ubuntu20.04-runner-ppc64le:2021-07-01", "entrypoint": [""] }
```

And then add `.power-e4s-generate-image` to the extends lists of the `e4s-on-power-pr-generate` and `e4s-on-power-develop-generate` jobs.